web/src/dialog_widget.ts

```diff
@@ -133,6 +134,8 @@ export function launch(conf: WidgetConfig): void {
     // * post_render: Callback to run after the modal body is added to DOM.
     // * loading_spinner: Whether to show a loading spinner inside the
     //   submit button when clicked.
+    // * update_submit_on_state_change: If true, updates state of submit button
+    //   on valid input change in modal
```

Comment:
This should end with a `.`, like all the others.

Reply:
Let's also call this `update_submit_disabled_state_on_change` to be super explicit what exactly is being updated.